src/inverseRobot.js

```diff
@@ -23,8 +23,19 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    if (robot[key] in result) {
+      return null;
+    } else {
```

Comment:
and you can get rid of this `else`

Reply:
I ridded "else", but the code became got longer, though more obvious